crates/consensus/common/src/validation.rs

```diff
@@ -307,6 +305,13 @@ pub fn validate_4844_header_standalone(header: &SealedHeader) -> Result<(), Cons
         })
     }
 
+    if excess_blob_gas % DATA_GAS_PER_BLOB != 0 {
+        return Err(ConsensusError::ExcessBlobGasNotMultipleOfBlobGasPerBlob {
+            excess_blob_gas,
+            blob_gas_per_blob: DATA_GAS_PER_BLOB,
+        })
+    }
```

Comment:
this indeed doesn't hurt and we should check this

could you please add your pr comment as comment here as well?

Reply:
Yes, will do 👍